test/tests/generic/test_crud_client.go

```diff
@@ -17,9 +17,11 @@ import (
 
 // Call within "It"
 func TestCrudClient(namespace1, namespace2 string, client ResourceClient, opts clients.WatchOpts, callbacks ...Callback) {
+	testOffset := 1
```

Comment:
Since this test is called within in `It`, we need to use ExpectWithOffset(1) so that errors are accurately displayed